src/Draco.Compiler/Internal/Documentation/DocumentationElement.cs

```diff
@@ -0,0 +1,25 @@
+using System.Collections.Immutable;
+using System.Linq;
+
+namespace Draco.Compiler.Internal.Documentation;
+
+internal abstract record class DocumentationElement
```

Comment:
When the PR becomes ready, let's document API. Until then no need, since this can change